reference-suite/src/main/courier/org/coursera/arrays/WithAnonymousUnionArray.courier

```diff
@@ -0,0 +1,6 @@
+namespace org.coursera.arrays
+
+record WithAnonymousUnionArray {
+  `unionsArray`: array[union[int, string]]
```

Comment:
Completely unnecessary...just didn't think to change it from the file I copied to make this test-case.

I'll update that in a followup pr...